--- fbida-2.14/GNUmakefile.orig	2018-05-28 21:14:02.388987847 +0200
+++ fbida-2.14/GNUmakefile	2018-05-28 21:19:24.712317500 +0200
@@ -67,11 +67,11 @@
 LIB		:= $(LIB)
 RESDIR		:= $(call ac_resdir)
 HAVE_LINUX_FB_H	:= $(call ac_header,linux/fb.h)
-HAVE_CAIRO_GL	:= $(call ac_pkg_config,cairo-gl)
-HAVE_LIBPCD	:= $(call ac_lib,pcd_open,pcd)
-HAVE_LIBGIF	:= $(call ac_lib,DGifOpenFileName,gif)
-HAVE_LIBWEBP	:= $(call ac_pkg_config,libwebp)
-HAVE_MOTIF	:= $(call ac_lib,XmStringGenerate,Xm,-L/usr/X11R6/$(LIB) -lXpm -lXt -lXext -lX11)
+HAVE_CAIRO_GL	?= $(call ac_pkg_config,cairo-gl)
+HAVE_LIBPCD	?= $(call ac_lib,pcd_open,pcd)
+HAVE_LIBGIF	?= $(call ac_lib,DGifOpenFileName,gif)
+HAVE_LIBWEBP	?= $(call ac_pkg_config,libwebp)
+HAVE_MOTIF	?= $(call ac_lib,XmStringGenerate,Xm,-L/usr/X11R6/$(LIB) -lXpm -lXt -lXext -lX11)
 JPEG_VER        := $(call ac_jpeg_ver)
 # deprecated
 #HAVE_GLIBC	:= $(call ac_func,fopencookie)
