--- fbida-2.07/rd/read-png.c.old	2010-01-31 09:53:29.000000000 +0100
+++ fbida-2.07/rd/read-png.c	2010-01-31 09:55:03.000000000 +0100
@@ -69,7 +69,7 @@ png_init(FILE *fp, char *filename, unsig
     if (h->color_type == PNG_COLOR_TYPE_PALETTE)
 	png_set_palette_to_rgb(h->png);
     if (h->color_type == PNG_COLOR_TYPE_GRAY && bit_depth < 8)
-	png_set_gray_1_2_4_to_8(h->png);
+	png_set_expand_gray_1_2_4_to_8(h->png);
 
     if (png_get_bKGD(h->png, h->info, &file_bg)) {
 	png_set_background(h->png,file_bg,PNG_BACKGROUND_GAMMA_FILE,1,1.0);
