diff -ur fbida-2.12/GNUmakefile fbida-2.12-cairo/GNUmakefile
--- fbida-2.12/GNUmakefile	2016-05-01 20:29:28.027653192 +0200
+++ fbida-2.12-cairo/GNUmakefile	2016-05-01 20:27:24.416185030 +0200
@@ -15,23 +15,8 @@
 PKGS_IDA := libexif
 PKGS_FBI := freetype2 fontconfig libdrm libexif
 PKGS_FBPDF := libdrm poppler-glib gbm epoxy cairo-gl
-HAVE_DEPS := $(shell $(PKG_CONFIG) $(PKGS_FBI) $(PKGS_FBPDF) && echo yes)
-
-ifeq ($(HAVE_LINUX_FB_H),yes)
-ifneq ($(HAVE_DEPS),yes)
-.PHONY: deps
-deps:
-	@echo "Build dependencies missing for fbi and/or fbpdf."
-	@echo "  fbi   needs:  $(PKGS_FBI)"
-	@echo "  fbpdf needs:  $(PKGS_FBPDF)"
-	@echo "Please install.  You can try 'make yum' (needs sudo)."
-	@false
-
-yum dnf:
-	sudo $@ install $(patsubst %,"pkgconfig(%)",$(PKGS_FBI) $(PKGS_FBPDF))
-
-endif
-endif
+HAVE_FBI_DEPS := $(shell $(PKG_CONFIG) $(PKGS_FBI) && echo yes)
+HAVE_FBPDF_DEPS := $(shell $(PKG_CONFIG) $(PKGS_FBPDF) && echo yes)
 
 # default target
 all: build
@@ -39,7 +24,13 @@
 # what to build
 TARGETS := exiftran thumbnail.cgi
 ifeq ($(HAVE_LINUX_FB_H),yes)
-  TARGETS += fbi fbpdf kbdtest
+ifeq ($(HAVE_FBI_DEPS),yes)
+  TARGETS += fbi
+endif
+ifeq ($(HAVE_FBPDF_DEPS),yes)
+  TARGETS += fbpdf
+endif
+  TARGETS += kbdtest
 endif
 ifeq ($(HAVE_MOTIF),yes)
   TARGETS += ida
@@ -248,10 +239,14 @@
 	$(INSTALL_BINARY) exiftran $(bindir)
 	$(INSTALL_DATA) $(srcdir)/exiftran.man $(mandir)/man1/exiftran.1
 ifeq ($(HAVE_LINUX_FB_H),yes)
+ifeq ($(HAVE_FBI_DEPS),yes)
 	$(INSTALL_BINARY) fbi $(bindir)
-	$(INSTALL_SCRIPT) fbgs $(bindir)
-	$(INSTALL_SCRIPT) fbpdf $(bindir)
 	$(INSTALL_DATA) $(srcdir)/fbi.man $(mandir)/man1/fbi.1
+endif
+ifeq ($(HAVE_FBPDF_DEPS),yes)
+	$(INSTALL_SCRIPT) fbpdf $(bindir)
+endif
+	$(INSTALL_SCRIPT) fbgs $(bindir)
 	$(INSTALL_DATA) $(srcdir)/fbgs.man $(mandir)/man1/fbgs.1
 endif
 ifeq ($(HAVE_MOTIF),yes)
